fineract-provider/src/main/java/org/apache/fineract/portfolio/loanproduct/serialization/LoanProductDataValidator.java

```diff
@@ -1964,6 +1968,49 @@ private void validateChargeToIncomeAccountMappings(final DataValidatorBuilder ba
         }
     }
 
+    private void validateChargeOffToExpenseMappings(final DataValidatorBuilder baseDataValidator, final JsonElement element) {
```

Comment:
Can we please validate whether existing charge off code value id was provided (existing charge off code value in DB) and existing expense type gl account id was provided?

Reply:
Done